.api-version

```diff
@@ -1 +1 @@
-5.0.9
\ No newline at end of file
+5.0.11
```

Comment:
We probably need to revert this back to 5.0.9, otherwise the API docs will be updated to 5.0.11.

Reply:
Noted. Updated in next commit.